llap-server/src/java/org/apache/hadoop/hive/llap/io/api/impl/LlapRecordReader.java

```diff
@@ -182,10 +183,15 @@ private LlapRecordReader(MapWork mapWork, JobConf job, FileSplit split,
 
     this.isVectorized = HiveConf.getBoolVar(jobConf, HiveConf.ConfVars.HIVE_VECTORIZATION_ENABLED);
     if (isAcidScan) {
+      OrcSplit orcSplit = (OrcSplit) split;
       this.acidReader = new VectorizedOrcAcidRowBatchReader(
-          (OrcSplit) split, jobConf, Reporter.NULL, null, rbCtx, true);
+          orcSplit, jobConf, Reporter.NULL, null, rbCtx, true);
+      isAcidFormat = !orcSplit.isOriginal();
+    } else {
+      isAcidFormat = false;
     }
-    this.includes = new IncludesImpl(tableIncludedCols, isAcidScan, rbCtx,
+
+    this.includes = new IncludesImpl(tableIncludedCols, isAcidScan && isAcidFormat, rbCtx,
```

Comment:
this can be simplified to `isAcidFormat ` instead of `isAcidScan && isAcidFormat`

Reply:
done